wapitiCore/attack/mod_https_redirect.py

```diff
@@ -50,12 +63,20 @@ async def attack(self, request: Request, response: Optional[Response] = None):
         url = urlparse(request.url)
 
         is_http = False
+        http_request = Request(
```

Comment:
You should be able to save a request by using the `response` parameter of the `attack()` function and directly checking its headers